src/components/aria-live/aria-live.spec.ts

```diff
@@ -0,0 +1,95 @@
+import {
```

Comment:
This may seem weird, but could you move this component into `core/` for now? I don't yet want to publish it as a first-class component.